kubemarine/resources/configurations/globals.yaml

```diff
@@ -108,43 +108,43 @@ compatibility_map:
       v1.21.2:
         version_rhel: 19.03*
         version_rhel8: 19.03*
-        version_debian: 5:19.03.*
+        version_debian: 5:20.10.*
```

Comment:
If we upgrade recommendend versions for existing clusters, we need to create patch to migrate the clusters

Reply:
New versions of packages will be used for new nodes, I don’t really see the point in creating a patch

Reply:
It is possible to run install --tasks prepare.cri or other task which in particular installs packages on existing clusters. There can be other reasons to run the tasks. In general patches are developed to make cluster consistent with the new version of kubemarine. So I suggest to emphasize that new version of kubemarine requires new packages, using How to Apply, and either automatic patch or manual steps.

Reply:
Add how to apply